CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ way to update this template, but currently, we follow a pattern:
 
 ---
 ## Upcoming version 2018-09-XX
+* [fix] obfuscatedCoordinatesImpl didn't always return coordinates within given
+  offset radius.
```

Comment:
Just a generic note on the abstraction level of this description: I think it is too detailed. Something like "Improved fuzzy location calculation" would probably suffice for people who read the changelog.